electrical/gps.py

```diff
@@ -4,13 +4,14 @@
 import time
 import pynmea2
 import csv
-if False: # import UbloxGPS if this is not a simulation
+from engine.is_raspberrypi import is_raspberrypi
+if is_raspberrypi():
     from ublox_gps import UbloxGps
 
 
 class GPS:
-    def __init__(self, init_serial, is_sim):
-        self.is_sim = is_sim
+    def __init__(self, init_serial):
+        self.is_sim = not is_raspberrypi()
```

Comment:
Oops yes! aayush brought this up and I forgot to change it to be passed down from robot_state (already has is_sim flag). Good catch o7

Also going to note here that we still need is_raspberrypi for the import statement since the flag set in the init statement